tests/integration/security/ca_custom_root/main_test.go

```diff
@@ -60,27 +51,21 @@ type EchoDeployments struct {
 }
 
 var (
-	inst istio.Instance
-	apps = &EchoDeployments{}
+	inst    istio.Instance
+	apps    deployment.SingleNamespaceView
+	echoApp = &EchoDeployments{}
 )
 
-func SetupApps(ctx resource.Context, apps *EchoDeployments) error {
+func SetupApps(ctx resource.Context, apps deployment.SingleNamespaceView) error {
```

Comment:
There should no longer be a custom apps in use. The idea is to use the common echo deployment. If the common deployment is missing things that we need here then we should consider adding them. That's what I did for the authz tests.